File: Csharp/Lab8/Lab8/Lab8/cng/Model/Collections/binaryTree/Node.cs
using Lab8.cng.Model.Entity.Abstracts;

namespace Lab8.cng.Model.Collections.binaryTree
{
    public class Node
    {
        private Toy data;
        private Node left;
        private Node right;

        public Node(Toy data)
        {
            this.data = data;
            left = null;
            right = null;
        }

        public Toy Data
        {
            get => data;
            set => data = value;
        }

        public Node Left
        {
            get => left;
            set => left = value;
        }

        public Node Right
        {
            get => right;
            set => right = value;
        }
    }
}

File: Java/Desktop/NewLab8/src/by/bntu/fitr/povt/cng/model/collections/neww/binaryTree/Node.java
package by.bntu.fitr.povt.cng.model.collections.neww.binaryTree;

import by.bntu.fitr.povt.cng.model.entity.abstracts.Toy;

class Node {

    private Toy getDate;
    private Node left;
    private Node right;

    public Node(Toy data){
        this.getDate = data;
        left = null;
        right = null;
    }

    public Toy getData() {
        return getDate;
    }

    public void setData(Toy data) {
        this.getDate = data;
    }

    public Node getLeft() {
        return left;
    }

    public void setLeft(Node left) {
        this.left = left;
    }

    public Node getRight() {
        return right;
    }

    public void setRight(Node right) {
        this.right = right;
    }
}
